#359 add empty check for applyBeforeCallback (#363)
commercetools/commercetools-sync-java: docs/RELEASE_NOTES.md

    - **ProductType Sync** - `ProductTypeSyncUtils#buildActions`, `ProductTypeUpdateActionUtils#buildAttributesUpdateActions`  
    now treat the values of the optional fields `isSearchable`, `inputHint` and `attributeConstraint` 
    as (`true`, `SingleLine` and `None` respectivley) if they are `null` or not passed. [#354](https://github.com/commercetools/commercetools-sync-java/issues/354)

>     - **Commons** - Fixed a bug in the `beforeUpdateCallback` which callback should not be called when there are no update actions. [#359](https://github.com/commercetools/commercetools-sync-java/issues/359)

Reviewer: ```suggestion
    - **Commons** - Fixed a bug in the `beforeUpdateCallback` which caused the callback to be called even on an empty list of update actions. [#359](https://github.com/commercetools/commercetools-sync-java/issues/359)
```
